pymatgen/io/cp2k/inputs.py

```diff
@@ -2089,7 +2089,7 @@ def __init__(self, npoints: int, kpoints: Iterable, units: str = "B_VECTOR") ->
         )
 
 
-@deprecated(KpointSet, "Kpoint_Set has been deprecated in favor of KpointSet on 2024-03-16")
+@deprecated(KpointSet, "Kpoint_Set has been deprecated in favor of KpointSet on 2024-03-16", deadline=(2025, 3, 16))
```

Comment:
I added an 1-year deadline here (consistent with its neighbors), free feel to change this.

Reply:
that works for me

Reply:
Great!